features/puzzle/puzzle-screen.html

```diff
@@ -61,24 +97,27 @@
 
 
         <div class="center">
-            <p>Countdown for 60 seconds: <span id="countdown"></span></p><!--Experimental timer 60s can be adjusted-->
-            <button onclick="countdown()">Start Timer</button>
+            <p>Countdown for 60 seconds: <span id="countdown"></span></p><!--New countdown timer-->
             <div class="input">
                 <input type="text" id="word" name="word" placeholder="Guess the Answer"><br><br>
-
-                <button class="button1">Submit</button>
-                <button class="button2">Pass</button>
+                <button class="button1" onclick="setUserInput()">Submit</button> <!--Buttons for game operations-->
+                <button class="button2" onclick="testCall()">Pass</button>
+                <button class="button3">Reset</button>
+                <button class="button4">Give Up</button>
```

Comment:
please come up with more readable names for the buttons.  E.x. submit-button/submit

also follow DRY in css. Instead of pasting the same styles on each button, make a generic button style then apply specific button related styles to button ids

Reply:
improved the names